### PR TITLE
More precise deps management for rpm build

### PR DESCRIPTION
**Pull request by bcornec**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,7 +8,8 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+requests>=2.9.1
 
 # Python3 compat
 future>=0.15.2
-configparser>=3.3.0; python_version < '3'
+#configparser>=3.3.0; python_version < '3'
```
Comment (uggla) [inline requirements.txt]: configparser is required and should not be commented for python2.
This issus is  "python_version < '3' " that need a really recent version of python setuptools.
I think that the current way to manage that is maybe to make a quick patch with sed in the .spec file.
So remove that "python_version < '3' " for python2 pkg and remove the complete line for python3 pkg.

Comment (bcornec) [inline requirements.txt]: The problem with messing up with this in the .spec is that we need to manage different cases depending of the version of setuptools adopted by the distribution. One way to solve that cleanly would be to use a PB macro in requirements.txt that will be instantiated based on the distro and thus create the right file. Or a macro in the spec doing this. I'll try that.

Comment (bcornec) [inline requirements.txt]: BTW for me the tool is working _without_ having a python-configparser tool !! So for which feature do we need it ?

Comment (bcornec) [inline requirements.txt]: It even doesn't exist in Mageia at all :-(

Comment (bcornec) [inline requirements.txt]: To be mor precise:

$ rpm -qf /usr/lib/python2.7/site-packages/configparser/**init**.py
python-future-0.15.2-1.mga5
$ rpm -qf /usr/lib64/python2.7/ConfigParser.py
python-2.7.9-2.mga5

So do we really need a separate module or is it part of python ?

Comment (uggla) [inline requirements.txt]: I thought about that. Mageia has embeded configparser into future.
But this is not the case for fedora and mainly pypi. (see below)

I'm not opposed to put PBMACRO into requirements.txt, but it means that you will have to generate the pypi stuff. So right now I would not go in this direction.
The sed trick is fine, using it we just manage to have a requirement.txt compatible with all the setuptools version and I think all distro . For 0.3 I will go for this.
More over this problem will not remain anymore as soons as distro will embed a recent setuptools.

```
[uggla@ugglalaptop ~]$ pip show configparser
You are using pip version 7.1.0, however version 8.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
---
Metadata-Version: 1.1
Name: configparser
Version: 3.5.0b2
Summary: This library brings the updated configparser from Python 3.5 to Python 2.6-3.5.
Home-page: http://docs.python.org/3/library/configparser.html
Author: Lukasz Langa
Author-email: lukasz@langa.pl
License: MIT
Location: /usr/lib/python2.7/site-packages
Requires: 

```

```
[uggla@ugglalaptop ~]$ rpm -qf /usr/lib/python2.7/site-packages/configparser.py
python-configparser-3.5.0b2-0.2.fc23.noarch
```

Comment (bcornec) [inline requirements.txt]: Today only fedora 23 provides a recent enough version of python-setuptools (18.0.1) to support the syntax python < '3'

**Pull request merged**